src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.272.0.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: Browser agent
+releaseDate: "2024-11-07"
+version: 1.272.0
+features: ["Marks and measures"]
+bugs: []
+security: []
+---
+
+## v1.272.0
+
+### Features
+
+#### Marks and measures
+Add detection for marks and measures which captures data as a `BrowserPerformance` event.  This feature is being shipped in an experimental state and is `off` by default. To opt in to using this feature, it can be turned on by setting the config value `init.performance.capture_marks = true` or `init.performance.capture_measures = true` in Copy/Paste or NPM installations.
```

Comment:
```suggestion
Add detection for marks and measures which captures data as a `BrowserPerformance` event. This feature is being shipped in an experimental state and is `off` by default. To opt in to using this feature, it can be turned on by setting the config value `init.performance.capture_marks = true` or `init.performance.capture_measures = true` in Copy/Paste or NPM installations.
```